lib/modules/manager/gradle/extract.spec.ts

```diff
@@ -33,6 +40,22 @@ describe('modules/manager/gradle/extract', () => {
     expect(res).toBeNull();
   });
 
+  it('logs a warning in case parseGradle throws an exception', async () => {
+    const filename = 'build.gradle';
+    const err = new Error('unknown');
+
+    jest.spyOn(parser, 'parseGradle').mockImplementationOnce(() => {
```

Comment:
can't we simply use mockRejectedValueOnce(err) to force async behavior?

Reply:
Sure, fixed in [7ff7222](https://github.com/renovatebot/renovate/pull/16030/commits/7ff722261724becb7f07d980a25ec6d7dd3aa704).